app/scripts/templates/sign_in_token_code.mustache

```diff
@@ -0,0 +1,31 @@
+<div id="main-content" class="card">
+  <header>
+    <h1 id="fxa-signin-code-header">{{#t}}Authorize this sign-in{{/t}}</h1>
+  </header>
+
+  <section>
+    <div class="error"></div>
+    <div class="success"></div>
+
+    <div class="graphic graphic-mail">{{#t}}Authorize this sign-in{{/t}}</div>
+
+    <p class="verification-email-message">
+    {{#t}}Check your email for the sign-in code sent to %(email)s.{{/t}}
```

Comment:
Note that user facing, the code is called `sign-in code`, which could get confused with the sign-in code used in our sms flows. Internally, I opted to call it `token code` since it verifies a token. Open to suggestions, or even dropping the word `signin` and displaying just code.

Reply:
Fwiw, I'm +1 to dropping the `sign-in` and just calling it a code in the front-end. Less is more when talking to users I reckon.